src/widget/sidebar.c

```diff
@@ -514,9 +324,14 @@ int widget_sidebar_handle_mouse_build_menu(const mouse *m)
     }
 }
 
+int widget_sidebar_get_expanded_width(void)
+{
+    return SIDEBAR_EXPANDED_WIDTH;
+}
+
```

Comment:
We should pass this data as parameter instead of `sidebar` calling `sidebar_filler`, and `sidebar_filler` having to call back to `sidebar` to get the width: such circular dependencies should be avoided.

Reply:
hmmm what about adding a `widgets/sidebar_constants.h` file for these kind of values?  while there may be some future where we have to use an x value that is different than SIDEBAR_EXPANDED_WIDTH, such a future seems unlikely, and i think it makes the code less clear to have sidebar_width passed around when it's purely the value of that constant.

Reply:
eh, i went ahead and removed the sidebar_constants.h after everything else was moved over, and added sidebar_width/height as params to sidebar_filler.  definitely cleaner, thanks again for the input :-) resolving